src/SimulationCSharpClient/Client/HttpRetryMessageHandler.cs

```diff
@@ -9,18 +9,25 @@ namespace SimulationCSharpClient.Client
 {
     public class HttpRetryMessageHandler : DelegatingHandler
     {
-        private readonly int maxRetries;
+        private const double ExponentialBackOff = 2.15;
 
         public HttpRetryMessageHandler()
             : base()
         {
-            this.maxRetries = 6;
+            this.MaxRetries = 6;
         }
 
         public HttpRetryMessageHandler(int maxRetries = 6, HttpClientHandler handler = null)
             : base(handler ?? new HttpClientHandler())
         {
-            this.maxRetries = (maxRetries <= 0 || maxRetries > 6) ? 6 : maxRetries;
+            this.MaxRetries = (maxRetries <= 0 || maxRetries > 6) ? 6 : maxRetries;
+        }
+
+        public int MaxRetries { get; private set; }
+
+        public double TotalWaitDurationInSecs
```

Comment:
reference all comments on workflow client PR

Reply:
@chadqueen Modified as per your comments on the Workflow-csharp-client PR